app/helpers/application_helper.rb

```diff
@@ -7,14 +7,21 @@ def content_tagger_url
     Plek.new.external_url_for("content-tagger")
   end
 
-  def preview_url(slug, auth_bypass_id: nil)
-    url = "#{Plek.new.external_url_for('draft-origin')}/#{slug.sub(/^\//, '')}"
-
-    if auth_bypass_id.present?
-      url = JwtHelper.access_limited_preview_url(url, auth_bypass_id)
-    end
+  def authenticated_preview_url(slug)
+    "#{Plek.new.external_url_for('draft-origin')}/#{slug.sub(/^\//, '')}"
```

Comment:
Do you know why we strip this first slash off the slug?

Is it new that we're having authenticated and anonymous preview urls? Most of our apps seem to consistently just give out the link with a token avoiding the need for two?

Reply:
Actually reading more I now understand. I see we're also using this function to link to other GOV.UK pages. That also explains the slash removal.

I think it might be clearer if this function was just `draft_govuk_url` and accepted a path rather than a slug then could be just `Plek.find("draft-origin", external: true) + path`

Arguably you could probably just have a helper for `draft_govuk` and then just append to paths in view code.

I do wonder if we might end up adding the token to some of the other urls at some point - but we can wait and see.

Reply:
> I think it might be clearer if this function was just draft_govuk_url and accepted a path rather than a slug then could be just Plek.find("draft-origin", external: true) + path

Thanks for the suggestion, I'll do this 👍